ubuntu LTSP box/Vagrantfile

```diff
@@ -19,18 +19,45 @@ end
 
 $LTSP_Installer_script = <<SCRIPT
 #!/bin/bash
```

Comment:
You can also separate the bash script from the Vagrantfile, like this example: https://github.com/Own-Mailbox/omb/blob/debian/Vagrantfile#L14
This would make more readable both the Vagrantfile and the installation/configuration script.

Reply:
okay that would be better. Done.